bfapi/service/jobs.py

```diff
@@ -208,7 +208,7 @@ def create(
         tide_max_24h=tide_max,
     )
 
-def __make_cli_cmd(
+def __make_algorithm_cli_cmd(
```

Comment:
It doesn't work that way for module-level functions, Ben.  Please check this gist:

https://gist.github.com/dbazile/c3a0eac7963ee36554c9961f0eae2f39